BUILDING.md

```diff
@@ -10,6 +10,7 @@ There is one special external dependency, Boost:
 * On other OSes, install Boost from your package manager. If you are on macOS and using [Homebrew](https://brew.sh/): `brew install boost`.
 
 Then you just need to run `cargo run --release` (for a release build) or `cargo run` (for a debug build) to build and run touchHLE. On an underpowered, passively-cooled, 2-core laptop (2017 Retina MacBook), a clean release build takes a bit less than 9 minutes.
+* To compile with dynamically linked libraries, append `--no-default-features` to the end of the cargo build command.
```

Comment:
Nitpick: this being a bullet point feels a bit out-of-place.

More importantly though, you should say which libraries are used (SDL2, OpenAL) and mention the consequence of this (you need to install them from your package manage).